src/tuneflow_py/models/protos/song_pb2.py

```diff
@@ -244,7 +244,7 @@
   serialized_end=2705,
 )
 _sym_db.RegisterEnumDescriptor(_STRUCTUREMARKER_STRUCTURETYPE)
-
+StructureType = enum_type_wrapper.EnumTypeWrapper(_STRUCTUREMARKER_STRUCTURETYPE)
```

Comment:
This is an auto-generated file, which will be overwritten every time the proto recompiles. We should not manually edit this file.